packages/wdio-browserstack-service/src/util.ts

```diff
@@ -97,7 +106,157 @@ export function getParentSuiteName(fullTitle: string, testSuiteTitle: string): s
     return parentSuiteName.trim()
 }
 
-export async function launchTestSession (options: BrowserstackConfig & Options.Testrunner, config: Options.Testrunner, bsConfig: UserConfig) {
+function filterPII(userConfig: Options.Testrunner) {
```

Comment:
What does PII mean? Can we have a comment what this function does?

Reply:
PII is user sensitive information present in config file which we filter out before sending config details. Will add the same as a comment.

Reply:
added comment